README.md

```diff
@@ -11,6 +11,34 @@ Source of truth for CDK code generation.
 - `@aws-cdk/service-spec-types` - loading, saving, querying and diffing a db
 - `@aws-cdk/aws-service-spec` - the actual db file
 
+## Data Sources
```

Comment:
Do we not use patches anymore?

Reply:
Fair enough.

Reply:
> Do we not use patches anymore?

Yes and no. But we need to describe that. 😅